l10n_es_aeat_mod190/views/account_fiscal_position.xml

```diff
@@ -6,7 +6,6 @@
         <field name="name">account.fiscal.position.form (in l10n_es_aeat_mod190)</field>
         <field name="model">account.fiscal.position</field>
         <field name="inherit_id" ref="account.view_account_position_form" />
-        <field name="groups_id" eval="[(4, ref('l10n_es_aeat.group_account_aeat'))]" />
```

Comment:
Esto debe ponerse a nivel de campos (todos los que tengan groups en la vista).

Reply:
Inherited view cannot have 'Groups' define on the record. Use 'groups' attributes inside the view definition